### ISSUE TITLE
Failed Sass/CSS Beautification, preserve_newlines ignored

### ISSUE THREAD

**Issue by mattalat** [Glavin001/atom-beautify#840]
### Description

Beautification seems to be ignoring supplied settings, specifically preserve_newlines and max_preserve_newlines, for my Sass (SCSS) and CSS code (using prettydiff). Received expected results when testing on prettydiff.com, however. 

Empty single-line comments in Sass are also being shuttled to the preceding line, while commented lines with content are collapsed. The empty single-line comment issue was not reproduced automatically in `debug.md`, but occurs when I enable `Beautify on save`.

### Results

Expected:

```
// Colors
$color: red; 

//
// Variables
// Don't include this comment in my output!
$myvar: 'iable';

/* Imports */
@import "imports/_variables";

/* Source */
.content {
  background-color: $color;
}

.text {
  color: black;

  a {
    text-align: center;
  }
}
```
Received:

```
// Colors
$color: red; // 
// Variables
// Don't include this comment in my output!
$myvar: 'iable';
/* Imports */
@import "imports/_variables";
/* Source */
.content {
  background-color: $color;
}

.text {
  color: black;

  a {
    text-align: center;
  }
}
```

### Steps to Reproduce

1. Reproducible on my machine in fresh project (with new Atom install)
2. Attempted using settings in both config.cson and .jsbeautifyrc

### Debug

`debug.md` gist: https://gist.github.com/mattalat/4aacf3c12db63ee15120

**Comment by prettydiff** [comment]
There was a very recent change in Pretty Diff to how empty lines are preserved, see https://github.com/prettydiff/prettydiff/issues/275

I can completely reproduce the defect.  I can reproduce the defect `$color: red; // ` outside Atom, but not the collapsed lines above the block comments.  I am investigating.

**Comment by prettydiff** [comment]
The noted defect of this line: `$color: red; //`, require a patch to Pretty Diff.  I resolved the rest by checking the option `CSS - Preserve newlines` in the Atom Beautify settings.

**Comment by mattalat** [comment]
Thank you for the quick response. I have a question, though: I've checked the `CSS - Preserve newlines` in the Atom Beautify settings, and I've redundantly enabled the setting using my project's `.jsbeautifyrc`, but I still don't see Atom Beautify preserving newlines in either my CSS or SCSS files. This is likely not an issue with prettydiff, as I can't recreate it online, should I open a new issue?

The odd thing is that checking the `HTML - Preserve newlines` seems to work just fine. However, for CSS/SCSS, the option is ignored.

**Comment by prettydiff** [comment]
No, please don't open an additional issue.  I will continue investigating and noting my findings here.  Let me update to the recently published .25 version and play around some more.  I will publish a resolution to the noted Pretty Diff defect in the next few hours.

**Comment by prettydiff** [comment]
I can confirm that I can reproduce the error in Atom on version .25 even with the option checked.

**Comment by prettydiff** [comment]
This is all a Pretty Diff defect.  Resolution published as v1.16.26.
